lib/pbench/test/unit/server/auth/test_auth.py

```diff
@@ -43,11 +43,12 @@ def fake_method(self, monkeypatch):
         args = {}
 
         def fake_method(
-            the_self, method: str, path: str, data: Dict, **kwargs
+            the_self, method: str, path: str, data: Dict, json: Dict, **kwargs
```

Comment:
OK, just skimming through randomly:

1. We don't need to use `typing.Dict` for 3.9+; `dict` would be better...
2. Any time we specify a `dict` or `list` in a type hint, we should really give the element types; e.g, `dict[str,str]`.

And, yeah, this is just a unit test mock ... it's still not a bad habit. 😆

Reply:
And, speaking of habits, we should be checking that the signature of the mock function is an appropriate match for the signature of the function being mocked....  😛